### ISSUE TITLE
Submit menu item

### ISSUE THREAD

**Issue by abhibeckert** [mmattozzi/cocoa-rest-client#116]
I'd like to add a new menu item for the `runSubmit:` action, and and set it's keyboard shortcut to `Command-R`.

I think the existing `reloadLastRequest:` menu item should be renamed to "Revert to Last Request" and have no keyboard shortcut.

Thoughts before I send through a pull request?

**Comment by mmattozzi** [comment]
Hmm, well I used `Command-R` to emulate the reload behavior that most people are familiar with using in their browser... although I guess there's no quick key combination to load a newly formed request if you don't happen to be editing the URL bar. Using the UI experience from Chrome, even if I change the URL in the location bar then hit `Command-R`, Chrome reloads the webpage that I am actually on, with whatever request parameters existed when I loaded the page. So this seems to match what cocoa-rest-client is doing. I'm not against adding some other shortcut for executing the current request which would be independent of where your focus is in the app.
